test/run_test.py

```diff
@@ -387,6 +387,12 @@ def run_test(
     env=None,
     print_log=True,
 ) -> int:
+    scribe_token = os.getenv("SCRIBE_GRAPHQL_ACCESS_TOKEN", "")
+    if scribe_token:
+        print_to_stderr("SCRIBE_GRAPHQL_ACCESS_TOKEN is set")
```

Comment:
Do you really want to print it to stderr? May be `log.info` here?

Reply:
run_test.py is just using print_to_stderr right now lol